.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,66 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [master]
+  schedule:
+    - cron: '0 17 * * 3'
```

Comment:
```suggestion
```

Reply:
But I think this was deliberate. It's better to check in on schedule rather than per-commit because the vulnerabilities are really appearing by someone adding new stuff but they are triggered by CVE being reported. I think weekly schedule for those is pretty good cadence.

Reply:
We will already have a higher cadence of commit though, so if anything we would catch it before that schedule

Reply:
I think it will just be overwhelming and counter-productive to get those reports for all commits, but Let's see :)

Reply:
I mean yeah I don't have a strong opinion on that one tbh. We can keep it too.

> I think it will just be overwhelming and counter-productive to get those reports for all commits, but Let's see :)

Wouldn't we get that if we keep "schedule" too, i.e. based on what we have in PR: we will currently have it when a PR is pushed, a commit is pushed and "on schedule" too, correct me if I am wrong though

Reply:
Yeah. I missed that when commenting on the comment. No strong opinion either, We need to see how the report looks like etc. I am ok with experimenting :)